.flake8

```diff
@@ -17,30 +17,13 @@ extend-ignore =
     E501,
     # Function definition does not bind loop variable, happens everywhere in our code
     B023,
-
-select =
-    # ===== built-in checks
-    # mccabe
-  # C901,
-    # pycodestyle errors
-  # E,
-    # pycodestyle warnings
-  # W,
-    # pyflakes
-  # F,
-    # ===== plugins
-    # flake8-builtins
-    A00,
-    # bugbear
-    B,
-    # flake8-comprehensions
-  # C4,
     # pydocstyle
-  # D,
+    D
     # flake8-rst-docstrings
-  # RST,
-    # flake8-broken-line
-  # N400
+    RST
```

Comment:
Do we want to ignore these?

Reply:
No ideally not. I did think about this, but for the repos I tested `D` generates a bunch of errors, but `RST` has only few. I am fine with removing `RST` and just address the issues that show up. However, for `D`, its probably going to be a lot of work addressing everything there. I see three options:
1. Keep ignoring `D`
2. Remove it and have everything break, which will force us to fix this. 
3. Do this in steps. For example, add `D2, D3, D4` to the ignore list, forcing us to fix all `D1xx` issues. Then later in another PR remove `D2` etc.

What do you think?

Reply:
Oh, I'm sorry, I got confused by the diff, I thought these were commented out from the `extend-ignore` field before. This is fine as is. We are doing some docstring linting with `flake8-docstrings` right?

Reply:
No, I think everything is and has been ignored for a while. `D` was not under `select` before, and is now under `extend-ignore` to keep the same behavior, so everything is ignored.

Reply:
Yeah those were all ignored because I couldn't be bothered to fix everything for every repo. They were aspirationally left in there so in the future we could maybe fix them up.